asciinema/scenario.sh

```diff
@@ -1,15 +1,11 @@
 echo "# See Tekton in action!" | pv -qL 12
```

Comment:
You'll also need to run the script to update `/static/asciinema/demo.cast` which is what actually gets displayed on the site.